host-controller/src/main/java/org/jboss/as/host/controller/HostControllerEnvironment.java

```diff
@@ -485,6 +487,7 @@ public HostControllerEnvironment(Map<String, String> hostSystemProperties, boole
         this.processType = processType;
 
         this.stability = getEnumProperty(hostSystemProperties, STABILITY, this.productConfig.getDefaultStability());
+        this.stabilities = Collections.unmodifiableSet(productConfig.getStabilitySet());
```

Comment:
Making this set unmodifiable is the right thing to do, but better to change that at the source: https://github.com/wildfly/wildfly-core/blob/main/version/src/main/java/org/jboss/as/version/ProductConfig.java#L97
That way every process env does not need to do the same wrapping.

Reply:
@pferraro Done